build-packages

```diff
@@ -72,14 +72,17 @@ launch_container() {
         lxc delete --force "$NAME"
     fi
 
-    lxc launch ubuntu-daily:"$release" "$NAME" ||
+    lxc launch ubuntu-daily:"$release" "$NAME" --ephemeral ||
```

Comment:
Hopefully nothing. But if `lxc delete --force` fails because of ZFS issues (or anything else), the stale STOPPED container will be auto-deleted at the next reboot.